pandas/tests/resample/test_deprecated.py

```diff
@@ -278,3 +278,36 @@ def test_resample_base_with_timedeltaindex():
 
     tm.assert_index_equal(without_base.index, exp_without_base)
     tm.assert_index_equal(with_base.index, exp_with_base)
+
+
+def test_interpolate_posargs_deprecation():
+    # GH 41485
+    df = DataFrame({"ds": ["1992-08-27 07:46:48", "1992-08-27 07:46:59"], "y": [1, 4]})
+    s = Series(
+        df.iloc[:, 1].values.reshape(-1), index=pd.to_datetime(df.iloc[:, 0].values)
+    )
+
+    msg = (
+        r"In a future version of pandas all arguments of DataFrame\.interpolate "
+        r"except for the argument 'method' will be keyword-only"
+    )
```

Comment:
Does this test pass? The message doesn't seem right, and I"d expect it to be different for the Series case

Have you run
```
pytest pandas/tests/resample/test_deprecated.py
````
locally?

Reply:
No I haven’t tested it. Will do it now

Reply:
OK, so that needs fixing first :) I suggest you remove the `with tm.assert_produces_warning`, dedent the following line, run the test, check what error message you get, and use that

Reply:
Done that, got new issues due to `resample("3s")`, trying to resolve it. I think I have to create `expected` in another way.